src/components/filter/filter.tsx

```diff
@@ -60,24 +44,14 @@ export const Filter: React.FC = () => {
 					</div>
 				</div>
 
-				<div className="flex flex-col gap-2 w-full">
-					<div className="font-bold text-xl">{i18n.filter.treeAge}</div>
-					<div className="flex flex-row gap-2">
-						{treeAgeIntervals.map((interval, index) => {
-							return (
-								<div key={`interval-${index}`} className="w-[33%]">
-									<TreeAgeButton
-										onChange={() => {
-											toggleTreeAgeInterval(interval);
-										}}
-										name={paddingMap[interval.identifier].label}
-										size={paddingMap[interval.identifier].size}
-										interval={interval}
-									/>
-								</div>
-							);
-						})}
-					</div>
+				<div className="flex flex-col w-full gap-y-6">
+					<AgeRangeSlider
+						min={0}
+						max={200}
+						onChange={({ min, max }: { min: number; max: number }) => {
+							setTreeAgeRange(min, max);
+						}}
```

Comment:
as commented further above, I think it would simpler to not have any props here 👍